pywinauto/application.py

```diff
@@ -912,6 +912,11 @@ def connect(self, **kwargs):
         if not connected:
             raise RuntimeError(
                 "You must specify one of process, handle or path")
+        else:
+            if 'path' not in kwargs:
+                if timeout is not None:
+                    message = "Timeout work only for call connect() with path"

```

Comment:
@vasily-v-ryabov, I thought about it, but what use case of this behavior? In other words, how to write a unit test, when we do not know the id before the process started?